lib/osf-components/addon/components/search-page/template.hbs

```diff
@@ -15,6 +17,30 @@ as |layout|>
             <p data-test-search-provider-description>
                 {{@provider.description}}
             </p>
+        {{else if @institution}}
+            <div data-test-heading-wrapper>
```

Comment:
We have two `data-test-heading-wrappers` in this file, one here and one in line7. We should probably remove this one and keep the one in line7

Reply:
Ok, that sounds good.